Dockerfile

```diff
@@ -1,50 +1,32 @@
-# For details and docs - see https://github.com/phusion/baseimage-docker#getting_started
-FROM phusion/baseimage:0.9.16
+FROM ubuntu:14.04
 
-# Use baseimage-docker's init system.
 CMD ["/sbin/my_init"]
 
-# Ensure UTF-8, required for add-apt-repository call.
-RUN locale-gen en_US.UTF-8
-ENV LANG       en_US.UTF-8
-ENV LC_ALL     en_US.UTF-8
+ENV HOME /root
 
-RUN add-apt-repository -y ppa:cz.nic-labs/bird && \
-    add-apt-repository -y ppa:project-calico/icehouse && \
-    apt-get update && \
-    apt-get install -qy \
-        calico-felix \
-        bird \
-        bird6 \
-        build-essential \
-        ipset \
-        iptables \
-        libffi-dev \
-        libssl-dev \
-        libyaml-dev \
-        python-dev \
-        python-docopt \
-        python-pip \
-        python-pyasn1 \
-        python-netaddr \
-        git \
-        python-gevent \
-        python-etcd \
-        && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ADD calico_containers/pycalico/requirements.txt /pycalico/
 
-# Confd
-RUN curl -L https://www.github.com/kelseyhightower/confd/releases/download/v0.9.0/confd-0.9.0-linux-amd64 -o confd && \
-    chmod +x confd
+# Incremental approach used for dev.
```

Comment:
@tomdee Expand these comments to explain.  E.g. "uncomment these lines and comment the 'prod' lines if you are doing lots of repeated builds, since the early scripts take a while and don't change often."